README.md

```diff
@@ -67,6 +67,10 @@ None
 
  * `postfix_smtpd_tls_cert_file` [default: `/etc/ssl/certs/ssl-cert-snakeoil.pem`]: Path to certificate file
  * `postfix_smtpd_tls_key_file` [default: `/etc/ssl/certs/ssl-cert-snakeoil.key`]: Path to key file
+ * `postfix_smtpd_tls_mandatory_protocols` [default: `SSLv2, !SSLv3, !TLSv1, !TLSv1.1`]:
+ * `postfix_smtpd_tls_protocols` [default: `SSLv2, !SSLv3, !TLSv1, !TLSv1.1`]:
+ * `postfix_smtpd_tls_mandatory_ciphers` [default: `medium`]:
```

Comment:
Not that I'm against adding this, but you could also use `postfix_raw_options`

Reply:
Ya, I did not notice that. you are right.
I pushed modifications about your quote comments but i will understand if the PR is not merged as i can use postfix_raw_options instead.